package.json

```
@@ -102,6 +102,7 @@
     "@types/lodash-es": "4.17.12",
     "@types/node": "20.11.0",
     "@types/platform": "1.3.6",
+    "@types/proper-lockfile": "^4.1.4",
```

please remove `^` we decided to use it like this cause it's a little bit more stable without it
```suggestion
    "@types/proper-lockfile": "4.1.4",
```